server/samples/clients/node/README.md

```diff
@@ -0,0 +1,60 @@
+# gRPC Client (Node) for Asherah Server
+A simple client application that demonstrates integrating with Asherah Server via a generated gRPC client.
+
+## Running the client
+Ensure the Asherah Server is running locally and listening on `unix:///tmp/appencryption.sock` and run:
+
+```console
+[user@machine node]$ node appencryption_client.js
+info: starting test
+info: encrypting payload BN3bOy1zCKQs
+info: received DRR
+info: decrypting DRR
+info: received decrypted data: BN3bOy1zCKQs
+info: test completed successfully
+
+```
+
+## Statically generating the client
+Install the client tools
```

Comment:
It's probably worth noting here that the sample client app doesn't support static cogegen. could also add a link to the static [routeguide example in the grpc repo](https://github.com/grpc/grpc/tree/v1.28.1/examples/node/static_codegen/route_guide)